autoload/hexokinase/patterns/color_names.vim

```diff
@@ -0,0 +1,155 @@
+let s:COLORS = {
```

Comment:
```suggestion
let s:colors = {
```

Reply:
Can the indentation for the colors be changed from 1tab+2spaces to 6 spaces (before the `\`).